index.coffee

```diff
@@ -32,7 +33,11 @@ module.exports = (grunt) ->
 
     {certificateFile, certificatePassword} = config
 
-    appMetadata = grunt.file.readJSON(path.join(appDirectory, 'resources', 'app', 'package.json'))
+    appResourcesDirectory = path.join(appDirectory, 'resources', 'app')
+    if grunt.file.exists(appResourcesDirectory) and grunt.file.isDir(appResourcesDirectory)
```

Comment:
I don't think you need to call exists here, `isDir` will only return true when it exists.